packages/component-library/stories/AccessibilityGuidelinesStyle.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';  
+import PropTypes from 'prop-types';
+// import { storiesOf } from '@storybook/react';
+// import { checkA11y } from '@storybook/addon-a11y';
+
+const styles = {
+  main: {
+    margin: 15,
+    maxWidth: 600,
+    lineHeight: 1.4,
+    fontFamily: '"Helvetica Neue", Helvetica, "Segoe UI", Arial, freesans, sans-serif',
+  },
+
+  logo: {
+    width: 200,
+  },
+
+  link: {
+    color: '#1474f3',
+    textDecoration: 'none',
+    borderBottom: '1px solid #1474f3',
+    paddingBottom: 2,
+  },
+
+  code: {
+    fontSize: 15,
+    fontWeight: 600,
+    padding: '2px 5px',
+    border: '1px solid #eae9e9',
+    borderRadius: 4,
+    backgroundColor: '#f3f2f2',
+    color: '#3a3a3a',
+  },
+};
+
+export default class AccessibilityGuidelines extends React.Component {

```

Comment:
I will do this.  Since the type of content (text, links, etc.) is similar to the Style pages, I copied the code from Welcome.js and modified it for the subject of that Style page, 

Should the Welcome.js page also be updated per your comments above?

Reply:
Just want to add... great way to clean up the code!

Reply:
@bradfordjanice Yes, updating the Welcome.js page would be great!